include/barcodes/qrcode.php

```diff
@@ -638,7 +638,8 @@ public function __construct($code, $eclevel = 'L') {
 		$barcode_array['bcode'] = array();
 		foreach ($qrTab as $line) {
 			$arrAdd = array();
-			foreach (str_split($line) as $char) {
+            $chars = function_exists('mb_str_split') ? mb_str_split($line) : str_split($line);
```

Comment:
Looks like incorrect indentation, project uses tabs while this code introduces spaces. Same can be said about all other new lines in this MR.